ibtk/include/ibtk/FEDataManager.h

```diff
@@ -635,11 +633,12 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
                                            double dx_min);
 
     /*!
-     * \brief Update the cell workload estimate. Does nothing unless a
-     * workload variable was previously registered via
-     * FEDataManager::registerLoadBalancer.
+     * \brief Update the cell workload estimate by adding a value (see the
+     * main documentation of this class for information on how this is
+     * computed) to the <code>d_workload_idx</code> cell variable.
      */
-    void updateWorkloadEstimates(int coarsest_ln = -1, int finest_ln = -1);
+    void addWorkloadEstimate(SAMRAI::tbox::Pointer<SAMRAI::hier::PatchHierarchy<NDIM> > hierarchy,
```

Comment:
"accumulate" or "account" instead of "add" here?

Reply:
Or maybe `addLocalWorkloadEstimate` or `addClassWorkloadEstimate`?

(`addWorkloadEstimate` sounds to me like we are telling the class about the workload estimate, rather than asking the class to account for its workload.)

Reply:
I like 'accumulate' since it implies mutability.

Reply:
Though, if you are willing to approve it as-is, I like 'add' more since it is shorter :imp:

Reply:
I think I will keep it as add. Using 'accumulate' here reminds me of `std::accumulate`, which does something different: in fact, `updateWorkloadEstimates` should really be `accumulateWorkloadEstimates` if we want to be more pedantic.